pyglmnet/pyglmnet.py

```diff
@@ -315,6 +325,19 @@ def _gradhess_logloss_1d(distr, xk, y, z, eta):
         hk = np.sum((y * _probit_g5(z, pdfz, cdfz) +
                      (1 - y) * _probit_g6(z, pdfz, cdfz)) * (xk * xk))
 
+    elif distr == 'neg-binomial':
+
+        r = 15.
+        mu = _mu(distr, z, eta)
+        grad_mu = _grad_mu(distr, z, eta)
+        hess_mu = np.exp(-z)*expit(z)**2
+
+        partial_beta_0_1 = hess_mu*((r+y)/(r+mu)-(y/mu))
```

Comment:
are you using a pep8 linter on your editor? This line should complain because you have missing whitespace around operators ...